documentation/docs/03-loading.md

```diff
@@ -111,6 +109,10 @@ For a route filename example like `src/routes/a/[b]/[...c]` and a `url.pathname`
 }
 ```
 
+#### props
+
+If the page you're loading has an endpoint, the data returned from it is accessible inside the leaf component's `load` function as `props`.
```

Comment:
Is it an empty object otherwise, or is it undefined?

Reply:
An empty object. will update

Reply:
:+1: Sounds good